code/game/gamemodes/clock_cult/clock_helpers/proselytizer_helpers.dm

```diff
@@ -49,6 +49,9 @@
 /turf/open/floor/clockwork/proselytize_vals(mob/living/user, obj/item/clockwork/clockwork_proselytizer/proselytizer, silent)
 	if(locate(/obj/structure/table) in src)
 		return FALSE
+	if(locate(/obj/structure/falsewall) in contents)
+		to_chat(user, "<span class='warning'>There is a false wall in the way, preventing you from proselytizing [src] into a clockwork wall.</span>")
```

Comment:
I swear I told you this after you suggested using [src] for the gang dominator name...